src/backend/distributed/utils/statistics_collection.c

```diff
@@ -0,0 +1,213 @@
+/*-------------------------------------------------------------------------
+ *
+ * statistics_collection.c
+ *	  Anonymous reports and statistics collection.
+ *
+ * Copyright (c) 2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#if HAVE_LIBCURL
+
+#include "postgres.h"
+
+#include <curl/curl.h>
+#include <sys/utsname.h>
+
+#include "access/xact.h"
+#include "citus_version.h"
+#include "distributed/metadata_cache.h"
+#include "distributed/statistics_collection.h"
+#include "distributed/worker_manager.h"
+#include "lib/stringinfo.h"
+#include "utils/json.h"
+
+bool EnableStatisticsCollection = true; /* send basic usage statistics to Citus */
+
+static uint64 NextPow2(uint64 n);
+static uint64 ClusterSize(List *distributedTableList);
+static bool SendHttpPostJsonRequest(const char *url, const char *postFields, long
+									timeout_seconds);
```

Comment:
Still have a snake-case variable name here (should be `timeoutSeconds`).

Reply:
Sorry for still missing this. Done.